html + css exercise/styles/style.scss

```diff
@@ -1,17 +1,25 @@
+/* Global Styles */
```

Comment:
Should I create a separate module for the variables? Or for the web / mobile views?